custom/reach/views.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from corehq.apps.domain.decorators import login_and_domain_required
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.locations.permissions import location_safe
+from django.utils.decorators import method_decorator
+from django.views.generic.base import TemplateView, View
+
+from custom.reach.const import INDICATOR_LIST, NUMERIC, PERCENT, COLORS
+
+
+class ReachDashboardView(TemplateView):
+    @property
+    def domain(self):
+        return self.kwargs['domain']
+
+    @property
+    def couch_user(self):
+        return self.request.couch_user
+
+    def get_context_data(self, **kwargs):
+        kwargs['domain'] = self.domain
+        # TODO add logic for user role type possible options should be MoHFW or MWCD
```

Comment:
When is the logic going to be added? For now I think we can decide that it will be in the user's custom data in a field called `ministry`

Reply:
Can you add a separate dropdown in this filter that allows web users to select between the hierarchies? This dropdown should be hidden fro mobile workers